src/Handlers/ShortcodeParametersHandler.php

```diff
@@ -0,0 +1,159 @@
+<?php
+
+namespace RebelCode\Bookings\WordPress\Module\Handlers;
+
+use Dhii\Exception\CreateInvalidArgumentExceptionCapableTrait;
+use Dhii\I18n\StringTranslatingTrait;
+use Dhii\Invocation\InvocableInterface;
+use Dhii\Storage\Resource\SelectCapableInterface;
+use Dhii\Transformer\TransformerInterface;
+use Psr\EventManager\EventInterface;
+use RebelCode\Expression\Builder\ExpressionBuilderInterface;
+use stdClass;
+
+/**
+ * Handler for rendering main application component's container.
```

Comment:
Some of the documentation in this class isn't making sense. Please consider revising.